lib/engine.py

```diff
@@ -199,27 +212,31 @@ def auto_get_datatypes(self, pk, source, columns, column_values):
                         value = values[i]
 
                         if self.table.cleanup.function != no_cleanup:
-                            value = self.table.cleanup.function(value, self.table.cleanup.args)
+                            value = self.table.cleanup.function(
+                                value, self.table.cleanup.args)
 
-                        if value != None and value != '':
+                        if value is not None and value is not '':
                             if len(str(value)) > max_lengths[i]:
                                 max_lengths[i] = len(str(value))
 
                             if column_types[i][0] in ('int', 'bigint'):
                                 try:
                                     value = int(value)
-                                    if column_types[i][0] == 'int' and hasattr(self, 'max_int') and value > self.max_int:
-                                        column_types[i] = ['bigint',]
+                                    if column_types[i][0] == 'int' and hasattr(self,
+                                                                               'max_int') and value > self.max_int:
```

Comment:
Since this only marginally helps the line length I'd just keep this all on one line.